cranelift/codegen/src/verifier/mod.rs

```diff
@@ -61,9 +61,9 @@ use crate::dbg::DisplayList;
 use crate::dominator_tree::DominatorTree;
 use crate::entity::SparseSet;
 use crate::flowgraph::{BlockPredecessor, ControlFlowGraph};
-use crate::ir;
 use crate::ir::entities::AnyEntity;
 use crate::ir::instructions::{BranchInfo, CallInfo, InstructionFormat, ResolvedConstraint};
+use crate::ir::{self, BlockWithArgs};
 use crate::ir::{
     types, ArgumentPurpose, Block, Constant, DynamicStackSlot, FuncRef, Function, GlobalValue,
```

Comment:
I've noticed rust-analyzer doesn't do quite what I want with imports like this. I'd want `BlockWithArgs` to go into the list of imports on line 68, and leave `use crate::ir;` as a separate item, but several times I've seen it do this kind of rewrite instead.

Reply:
What about adding a `self` import on line 67 and removing line 66?

Reply:
I don't feel strongly either way about using `self` imports. I just don't like having two import items with different parts of the import list like this.

I'd actually be tempted to quit importing anything except `ir` and `ir::types`, and qualify every reference to names in the `ir` module, given how short the module name is. But that's not something to do in this PR; it has enough else going on already.